bundler.d/windows.rb

```diff
@@ -2,4 +2,5 @@
   gem 'highline', :platform => :mingw
   gem 'win32-service', :platforms => :mingw
   gem "win32-open3", :platforms => :mingw_18
+  gem "open3", :platforms => :mingw_20
```

Comment:
You don't need this line, ```open3``` is available in the baseline ruby starting with 1.9.3 (and there's no gem called 'open3'. Chances are you are on x64_mingw_20, that's why bundler didn't complain).